### ISSUE TITLE
Add "Shelf" column in book list

### ISSUE THREAD

**Issue by placoderm** [bookwyrm-social/bookwyrm#2484]
**Is your feature request related to a problem? Please describe.**
When looking at your list of books, there is no indication what shelf a book is on. 

**Describe the solution you'd like**
Add a column that indicates the shelf the book is on.

**Describe alternatives you've considered**
I know you can filter by shelf, but that's not what I am suggesting. Filtering does not tell me what shelf a book is on, except in a cumbersome way of filtering each shelf till I find the book.

**Additional context**

![2022-12-09 15_37_13-Your books - bookrastinating com — Mozilla Firefox](https://user-images.githubusercontent.com/31570884/206678493-ba601821-2d86-4722-9cd0-5f9bd11f1345.png)

Apologies if this feature already exists. If it does exist, then it should be more prominent.

**Comment by marcusyoung** [comment]
Not what you are suggesting, but if you click on the Move Book drop down on the right you can see where the book is currently shelved. Having a shelf column would be good. It would need to be able to deal with books that are on multiple shelves though (from a layout point of view).

**Comment by placoderm** [comment]
Right, that's a workaround. 

But *can* books be on more than one shelf? I just tried moving a book to a different shelf and it removed it from a previous one. At least I think that is what happened. It's actually kind of confusing. 

But yes, indeed, if books can be on more than one shelf the column would have to indicate all of them.

**Comment by marcusyoung** [comment]
Not with the built-in shelves I don't think. But if you move a book to a custom shelf it isn't removed from, e.g the reading shelf.  Whether this is intended or a bug with the move functionality I'm not sure!

**Comment by placoderm** [comment]
Right. I have explored a bit more of the move thing (which, for the record, is not directly related to this issue) and indeed "moving" a book to another custom shelf doesn't move it, it just adds it to the shelf.

See also this issue I just created if you are interested: https://github.com/bookwyrm-social/bookwyrm/issues/2485
